addons/cockpit/functions/fnc_processSpeed.sqf

```diff
@@ -15,8 +15,8 @@ if (EGVAR(main,hasACEUnits)) exitWith {
 };
 
 private _output = switch (GVAR(checklistUnits)) do {
-	case ("KIAS"): {_kias + " KIAS"};
-	case ("KM/H"): {_kph + " KM/H"};
+  case "KIAS": { (_kias toFixed 0) + " KIAS" };
+  case "KM/H": { (_kph toFixed 0) + " KM/H" };
```

Comment:
Any reason for 2-space indentation & spaces within code block?

Reply:
I wish to have coding style unified with other files.
e.i. https://github.com/mgkid3310/AWESome/blob/96df7be770d8a36d2408b0ec1b24bcd4c00573c4/addons/gpws/functions/fnc_flightPhaseCheck.sqf#L14